services/thoth-cron/Dockerfile

```
@@ -0,0 +1,30 @@
+WORKDIR /app
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends curl && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY services/thoth-cron /app
+
+ENV PYTHONPATH=/app/src \
+    HOME=/home/appuser
+
+RUN pip install --no-cache-dir --upgrade pip && \
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)